docs/blender_docs/io_gltf2.rst

```diff
@@ -30,7 +30,198 @@ This importer/exporter supports the following glTF 2.0 features:
+
+These values are read from the "Principled BSDF" node.  If both of these inputs
+are disconnected, the node will display sliders to control their respective
+values between 0.0 and 1.0, and these values will be copied into the glTF.
+
+When using an image, glTF expects the metallic values to be encoded in the
+Blue (``B``) channel, and roughness to be encoded in the Green (``G``) channel of the
+same image.  If images are connected to the Blender node in a manner that
+does not follow this convention, this addon may attempt to adapt the image
+to the correct form during export, costing some performance.
+
+In the Blender node graph, it is recommended to use a "Separate RGB" node
+to separate the channels from an "Image Texture" node, and connect the
+Green (``G``) channel to Roughness, and Blue (``B``) to Metallic.  The glTF exporter
+will recognize this arrangement as matching the glTF standard, and that will
+allow it to simply copy the image texture into the glTF file during export.
+
+The Image Texture node for this should have its "Colorspace" setting
+configured to "Non-Color Data".
+
+.. figure:: /images/addons_io-gltf2-material-metalRough.png
+   :alt: A metallic-roughness image connected in a manner consistent
+         with the glTF standard, allowing it to be used verbatim inside
+         an exported glTF file.
+
+   A metallic/roughness image connected in a manner consistent
+   with the glTF standard, allowing it to be used verbatim inside
+   an exported glTF file.
+
+Baked Ambient Occlusion
+^^^^^^^^^^^^^^^^^^^^^^^
+
+glTF is capable of storing a baked ambient occlusion map.  Currently there
+is no arrangement of nodes that causes Blender to use such a map in exactly
+the same way as glTF intends it to be used.  However, if the exporter finds
+a custom node group by the name of ``glTF Metallic Roughness``, and finds an
+input named ``Occlusion`` on that node group, it will look for an Image Texture
+attached there to use as the occlusion map in glTF.  The effect need not be shown
+in Blender, as Blender has other ways of showing ambient occlusion, but this
+method will allow the exporter to write an occlusion image to the glTF.
+
+glTF stores occlusion in the Red (``R``) channel, allowing it to optionally share
+the same image with the Roughness and Metallic channels.
+
+Normal Map
+^^^^^^^^^^
+
+To use a Normal Map in glTF, connect an "Image Texture" node's color output
+to a "Normal Map" node's color input, and then connect the "Normal Map" normal
+output to the "Principled BSDF" node's "Normal" input.  The Image Texture node
+for this should have its "Colorspace" setting configured to "Non-Color Data".
+
+The "Normal Map" node must remain on its default setting of "Tangent Space" as
+this is the only type of normal map currently supported by glTF.  The strength
+of the normal map can be adjusted on this node.  The exporter isn't exporting
+these nodes directly, but will use them to locate the correct image and will
```

Comment:
This last sentence (`The exporter isn't exporting these nodes directly...`) seems unnecessary with the disclaimer at the top of the manual.

Reply:
Yeah, but I think I just removed that...